README.md

```diff
@@ -43,12 +43,13 @@ how to spend its XTZ and tokens.
 
 ## Deploying/Originating
 
-This contract is too large to fit inside the current Tezos origination limit.
+The contract can be originated with `tezos-client`:
 
-To overcome this limitation we advice to use the [morley-large-originator](https://gitlab.com/morley-framework/morley/-/tree/master/code/morley-large-originator)
-tool or the workaround that is described there.
+```sh
+tezos-client originate contract baseDAO transferring 0 from alice running BaseDAO.tz --init "$(<storage)" --burn-cap 17.0
+```
 
-For more information about this follow the [origination guide](docs/originating.md).
+Instructions on how to install `tezos-client` can be found at official [Tezos installation page](http://tezos.gitlab.io/introduction/howtoget.html) or in [our `tezos-packaging` repository](https://github.com/serokell/tezos-packaging).
```

Comment:
@pasqu4le Took me a while to set up `tezos-client` with the new protocol to test if it works. The size now increase from 16kB to 32kB which allows originating baseDAO now (baseDAO size is 18kB).

Related to README.md, I'm not sure what words to put in here, so I just used what was written before and put it back in. Let me know if this is good enough.

Reply:
> baseDAO size is 18kB

That's the contract size on its own, right?
Did you try to originate the various instantiation (`registryDAO` and `trivialDAO`)?

I ask because the initial storage also has to be taken into account in the origination and in baseDAO with the lambdas it can be quite big (I don't think it will be big enough to reach the limit tho, but I'm just checking).

Reply:
> Related to README.md

Just checked it, I think it's fine the way you changed it :ok_hand:

Reply:
> That's the contract size on its own, right?

I just realize that it is only with the trivialDao storage. With registryDAO and treasuryDAO storage, they are around 26kB and 23kB respectively. 

Definitely not far from the max size and it could be a concern in the future.

Reply:
I think that's plenty of margin however.

Some simple contracts can entirely fit inside the difference there :sweat_smile: 

This should be more than enough for custom DAOs and the contract implementation is not supposed to grow anymore, so this is ok :ok_hand: